### ISSUE TITLE
Use a distinguished icon for browser launched from Zap

### ISSUE THREAD

**Issue by tko** [zaproxy/zaproxy#8567]
### Is your feature request related to a problem? Please describe.

When launching Firefox from ZAP its icon in the Cmd-Tab view is identical to the Firefox I use for everything else making it a 50-50 chance for me to pick the one I was going for.

### Describe the solution you'd like

Using an icon that is in any way different from the default Firefox icon would make it much easier pick the right app.

A big Zap icon overlaying one quarter of the Firefox icon should do the trick.

### Describe alternatives you've considered

Guess I could use Chrome or Safari instead, I prefer not to.

### Screenshots

<img width="304" alt="SCR-20240720-svcv-2" src="https://github.com/user-attachments/assets/23e098bf-1faf-488f-ae80-ed8ffba60c0c">

One of them is from Zap.

### Would you like to help fix this issue?

- [ ] Yes

**Comment by kingthorin** [comment]
I don't think we have any control over it. It's just launch Firefox (or whatever) from the system.

**Comment by tko** [comment]
The proxy settings, among other things are different, no way to affect the icon? (Which is different while downloading something IIRC)

**Comment by psiinon** [comment]
I'm also not aware of any way we can change the icon of an app we launch from ZAP.
I can see why it would be useful, but I just dont think we can do it.

**Comment by tko** [comment]
Oh, I must've been thinking brew build or something. I see it's not exactly supported as command line option.

On macos I think you can manually make an alias and assign a custom icon to the alias. No idea how to do that programmatically.
